tests/javascripts/fileUpload.test.js

```diff
@@ -106,7 +106,7 @@ describe('File upload', () => {
 
     var buttonLabel;
 
-    uploadLabel.innerHTML += '<span class="error-message">PNG images only!</span>';
+    uploadLabel.innerHTML += '<span class="error-message">Logo must be a PNG file</span>';
```

Comment:
```suggestion
    uploadLabel.innerHTML += '<span class="error-message">The logo must be a PNG file</span>';
```

For consistency with the other instances of this field.